charts/astronomer/templates/registry/registry-networkpolicy.yaml

```diff
@@ -21,6 +21,16 @@ spec:
   - Ingress
   ingress:
   - from:
+    - podSelector:
+        matchLabels:
+          tier: astronomer
+          component: houston-update-check
+          release: {{ .Release.Name }}
+    - podSelector:
+        matchLabels:
+          tier: astronomer
+          component: houston-expire-deployments
+          release: {{ .Release.Name }}
```

Comment:
what about `houston-cleanup`?

Reply:
@sjmiller609 do you mind to add https://github.com/astronomer/astronomer/blob/b8e083fd0cd7fac110e781174d1b5f9b899a0504/charts/astronomer/templates/houston/cronjobs/houston-cleanup-deployments-cronjob.yaml#L26 too?

Reply:
hey actually I think it's down a few lines in this file

Reply:
line 52